### PR TITLE
3.27.0 docs - deploy with release 9/16

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

**Comment by Bucimis** [comment]
@KellieHawks ready for merge :)

File: _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ As a result, we recommend continuing to collect the IDFA if you meet any of the
...
+Apple has temporarily reverted the proposed per-app IDFA access change in iOS 14. For now prompting for permission with `AppTrackingTransparency` is not required, but you should be prepared for a future release from Apple that will require it to be implemented. When the `AppTrackingTransparency` prompt is required, in addition to implementing Braze's `ABKIDFADelegate` protocol, your application will need to request authorization using Apple's `ATTrackingManager` in the App Tracking Transparency framework. For more information, please reference this [Braze iOS 14 guide](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/ios_14/#idfa-and-app-tracking-transparency), [Apple's Overview](https://developer.apple.com/app-store/user-privacy-and-data-use/), and [Apple's Developer Documentation](https://developer.apple.com/documentation/apptrackingtransparency). In iOS 14, collecting IDFA will require building with Xcode 12, collecting IDFA will Xcode 11 is not possible in iOS 14.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations.md:47]:
```suggestion
Apple has temporarily reverted the proposed per-app IDFA access change in iOS 14. For now, prompting for permission with `AppTrackingTransparency` is not required, but you should be prepared for a future release from Apple that will require it to be implemented. When the `AppTrackingTransparency` prompt is required, in addition to implementing Braze's `ABKIDFADelegate` protocol, your application will need to request authorization using Apple's `ATTrackingManager` in the App Tracking Transparency framework. For more information, please reference this [Braze iOS 14 guide](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/ios_14/#idfa-and-app-tracking-transparency), [Apple's Overview](https://developer.apple.com/app-store/user-privacy-and-data-use/), and [Apple's Developer Documentation](https://developer.apple.com/documentation/apptrackingtransparency). In iOS 14, collecting IDFA will require building with Xcode 12, collecting IDFA will Xcode 11 is not possible in iOS 14.
```

File: _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ As a result, we recommend continuing to collect the IDFA if you meet any of the
...
+Apple has temporarily reverted the proposed per-app IDFA access change in iOS 14. For now, prompting for permission with `AppTrackingTransparency` is not required, but you should be prepared for a future release from Apple that will require it to be implemented. When the `AppTrackingTransparency` prompt is required, in addition to implementing Braze's `ABKIDFADelegate` protocol, your application will need to request authorization using Apple's `ATTrackingManager` in the App Tracking Transparency framework. For more information, please reference this [Braze iOS 14 guide](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/ios_14/#idfa-and-app-tracking-transparency), [Apple's Overview](https://developer.apple.com/app-store/user-privacy-and-data-use/), and [Apple's Developer Documentation](https://developer.apple.com/documentation/apptrackingtransparency). In iOS 14, collecting IDFA will require building with Xcode 12, collecting IDFA will Xcode 11 is not possible in iOS 14.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations.md:47]:
```suggestion
Apple has temporarily reverted the proposed per-app IDFA access change in iOS 14 until 2021. For now, prompting for permission with `AppTrackingTransparency` is not required, but you should be prepared for a future release from Apple which will require it to be implemented. 

When the `AppTrackingTransparency` prompt is required, in addition to implementing Braze's `ABKIDFADelegate` protocol, your application will need to request authorization using Apple's `ATTrackingManager` in the App Tracking Transparency framework. For more information, please reference this [Braze iOS 14 guide](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/ios_14/#idfa-and-app-tracking-transparency), [Apple's Overview](https://developer.apple.com/app-store/user-privacy-and-data-use/), and [Apple's Developer Documentation](https://developer.apple.com/documentation/apptrackingtransparency). In iOS 14, collecting IDFA will require building with Xcode 12, collecting IDFA will Xcode 11 is not possible in iOS 14.
```

**Review by davidbielik — DISMISSED** [review]

**Review by KellieHawks — APPROVED** [review]
lgtm! @davidbielik, let me know when you have finished reviewing and are ready to have this merged.

**Pull request merged**